src/Action/AddAction.php

```diff
@@ -166,4 +169,18 @@ protected function _error(Subject $subject)
         $this->setFlash('error', $subject);
         $this->_trigger('beforeRender', $subject);
     }
+
+    /**
+     * Stopped callback
+     *
+     * @param \Crud\Event\Subject $subject Event subject
+     * @return \Cake\Network\Response
+     */
+    protected function _stopped(Subject $subject)
+    {
+        $subject->set(['success' => false]);
```

Comment:
You have a very good point there mr. @jippi and yes, success would definitely be a most valid outcome. Would the following come close? My brain is a bit fried atm.

```php
        $success = false;
        if (!empty($subject->success)) {
            $success = $subject->success;
        }

        $subject->set(['success' => $success]);
```